README.md

```diff
@@ -32,16 +32,20 @@ Google Groups, IRC #activeadmin and Gitter are not actively monitored.
 
 ## Want to contribute?
 
-The [contributing guide][contributing]
-is a good place to start. If you have questions, feel free to ask.
+If you want to contribute through code or documentation, the [contributing
+guide][contributing] is a good place to start. If you have questions, feel free
+to ask.
 
-## Want to support us?
+If you want to contribute economically, you can help funding the project
```

Comment:
I felt it was better using the word "support" but its not a necessary change. Although once I did that I realized "economically" sounds weird, it's much more common to use the word financial when using support.

```suggestion
If you want to support us financially, you can help fund the project
```